src/content/pages/accessibility.mdx

```diff
@@ -120,14 +120,14 @@ Registration will be on the **1st floor**
 ### Low stimulation room (South room 243)
 
 - If you feel overwhelmed and need a quiet space with no distraction feel free to visit **South Room 243** on the **2nd floor**. To get there look for childcare and keep on going through the hallway. It will be one of the doors to your left.
+- You'll find a bag with sunglasses, eyemasks and earplugs for your convenience there. Please return the sunglasses and eyemasks back. Earplugs are yours forever.
 
-### Neurodiversity box
+### Neurodiversity bag
 
-We know that some conference participants can feel overwhelmed by large crowds. To help with this, we've prepared a box full of squishy and plushy items for you to relax during the demanding conference days.
-It is placed in the [Quiet Room](#quiet-room-meeting-22). Feel free to play with anything you like and please bring it back for others to enjoy as well
-(except for the earplugs, those are yours to have :)).
+We know that some conference participants can feel overwhelmed by large crowds. To help with this, we've prepared a bag full of squishy and plushy items for you to relax during the demanding conference days.
```

Comment:
Some people might feel that we don't really "know" their experience but appreciate that our understanding. 
```suggestion
We understand that some conference participants can feel overwhelmed by large crowds. To help with this, we've prepared a bag full of squishy and plushy items for you to relax during the demanding conference days.
```